hardware_applications/week5/final_main.cpp

```diff
@@ -0,0 +1,63 @@
+#include <STSL/RJRobot.h>
```

Comment:
Can we remove this file?

Reply:
... If we do... the instructors will not have the code they actually need to guide people through? Unless I'm misunderstanding your request.

Reply:
you understand me correctly

Reply:
See longer comment on the main thread, written after I saw your longer comment in the main conversation thread.